src/composables/usePool.ts

```diff
@@ -643,6 +643,10 @@ export function usePool(pool: Ref<AnyPool> | Ref<undefined>) {
       )
   );
 
+  const isGreatMigratablePool = computed(() => {
```

Comment:
Since more pools could be treated exactly like this in the future lets rename this to be more generic.
```suggestion
  const isDeprecatedPool = computed(() => {
```

Reply:
fixed